server/src/models/example.model.js

```diff
@@ -1,30 +0,0 @@
-const mongoose = require("mongoose");
-const Schema = mongoose.Schema;
-
-const exampleSchema = new Schema(
-  {
-    name: {
-      type: String,
-      required: true
-    },
-    description: {
-      type: String,
-      required: true
-    },
-    duration: {
-      type: Number,
-      required: true
-    },
-    date: {
-      type: Date,
-      required: true
-    },
-  },
-  {
-    timestamps: true
-  }
-);
-
-const Example = mongoose.model("Example", exampleSchema);
-
-module.exports = Example;
```

Comment:
Kindly don't delete this as this for reference of how to create a model and what conventions to inherit.

Reply:
okay, okay I will not delete the files which will use for references in the future